### PR TITLE
#161158698 Fix Sequelize error when trying to create guesthouse

### PR DESCRIPTION
**Pull request by amarachukwu-agbo**
#### What does this PR do?
Fixes the error when a travel admin tries to create a guesthouse

#### Description of Task to be completed?
Increase bedId numbers in the seeders files to large numbers

#### How should this be manually tested?
 - Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
 - Enter into the project directory - `cd travel_tool_back`
 - Start the server - `make start`
 - Navigate to [travela](http://travela-local.andela.com:3000/#) in your browser and log in
 - Login on your browser and grab your token from the application tab in Developers tab
 - Open Postman
 - Add your token in headers
 - click on the body option, select raw and choose JSON from the drop down
 -  use the example below for the body of the POST request 
'{
	"houseName": "Small Lagos ",
	"location": "Lagos Nigeria",
	"bathRooms": "5",
	"imageUrl": "https://www.lol.com",
	"rooms": [
		{
			"roomName": "big cutter",
			"roomType": "ensuited",
			"bedCount": "2"
		},
		{
			"roomName": "small cutter",
			"roomType": "non-ensuited",
			"bedCount": "1"
		}
	]
}' 

 - send a post request to the endpoint using postman `/api/v1/guesthouses`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161158698](https://www.pivotaltracker.com/story/show/161158698)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: src/database/seeders/20181004171126-demo-trips.js [hunk 1]
```diff
@@ -50,10 +50,10 @@ module.exports = {
         departureDate: '2018-10-16',
         returnDate: '2018-10-20',
         checkStatus: 'Not Checked In',
-        notificationCount: 0,
+        notificationCount: 6003,
```
Comment (samuelkubai) [inline src/database/seeders/20181004171126-demo-trips.js]: Why this change?

**Comment by amarachukwu-agbo** [comment]
@samuelkubai , mistakenly changed that. Fixed
